NewForce-Cohort-4/nutshell-team-odd-eclipse
# Ac task date

# Description
Changed the delete feature to a completed check box that will mark the task as complete and remove it from listing on the Dom.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.


# Testing Instructions
select enter a new task, fill out the form,  the task appears a blank check box is there too, when the blank check box is checked the task will remove the completed task
# Checklist:
- [ X] My code follows the style guidelines of this project
- [ X] I have performed a self-review of my own code
- [ X] I have commented my code, particularly in hard-to-understand areas
- [ X] My changes generate no new warnings or errors
- [ X] I have added test instructions that prove my fix is effective or that my feature works
